src/platform/forms/save-in-progress/SaveStatus.jsx

```diff
@@ -18,13 +18,22 @@ function SaveStatus({
   let savedAtMessage;
   if (lastSavedDate) {
     const savedAt = moment(lastSavedDate);
-    savedAtMessage = ` Last saved at ${savedAt.format(
+    savedAtMessage = ` It was last saved on ${savedAt.format(
       'MMM D, YYYY [at] h:mm a',
     )}`;
   } else {
     savedAtMessage = '';
   }
 
+  const formId = loadedData?.metadata?.inProgressFormId;
+  const formIdMessage =
+    formId && savedAtMessage ? (
+      <>
+        {' '}
+        Your application ID number is <strong>{formId}</strong>.
```

Comment:
:confused: I see `ID` here, but `Id` in your screenshot [here](https://github.com/department-of-veterans-affairs/vets-website/pull/14986/files#r526461304). What's the story?

Reply:
The last "mis-aligned" screenshot is me editing the content in the devtools... I forgot to capitalize the "D" in it... if you look in the code, it's "ID"